ambry-api/src/main/java/com.github.ambry/config/ReplicationConfig.java

```diff
@@ -106,6 +106,13 @@
   @Default("true")
   public final boolean replicationIncludeAll;
 
+  /**
+   * If true, replication token will be persisted explicitly.
+   */
+  @Config("replication.persist.token.on.shutdown.or.replica.remove")
```

Comment:
Could you make the comment clearer? I am trying to understand difference of `shutdown` and `replica move`. Does "token will be persisted explicitly" mean clean shutdown scenario?

Reply:
For CloudBackupManger and ReplicationManager, token is persisted when shutdown.
For CloudBackupManger, replica may be removed, token is also persisted for this case.
ReplicationManager doesn't support replica remove now.
This is used for test only as of now. 

I can add above into code comment.